docs/developing.rst

```diff
@@ -89,6 +89,12 @@ Then back in your host shell::
     pip install -e .[server]
     python -m mockprock.server
 
+If you use Z shell (zsh), the command ``pip install -e .[server]`` will fail with ``zsh: no matches found: .[server]``. This is because `zsh uses square brackets for globbing/pattern matching`_. You should instead run the following command.::
+
+   pip install -e ".[server]"
+
+.. _"zsh uses square brackets for globbing/pattern matching": https://stackoverflow.com/questions/30539798/zsh-no-matches-found-requestssecurity
+
```

Comment:
Something tells me that Catalina will lead to helpful notes like this being peppered throughout our docs :stuck_out_tongue: